package.json

```diff
@@ -69,5 +69,10 @@
             }
         ]
     },
+    "files": [
+        "lib",
+        "dist",
```

Comment:
probably `dist/near-api-js.min.js` eh?

Reply:
We decided to keep the not-minified version for now.